test/lsp-javascript-test.el

```diff
@@ -105,42 +105,4 @@
   (should (not (lsp-typescript-javascript-tsx-jsx-activate-p "abc.tsxx")))
   (should (not (lsp-typescript-javascript-tsx-jsx-activate-p "abc.jss"))))
 
-(ert-deftest lsp-clients-extract-signature-from-clangd-on-hover ()
```

Comment:
I believe these tests should not be removed, right?

Reply:
Good catch. lsp-clients-test.el has to still hold clangd tests!